src/app/[locale]/t/[bookId]/_components/reader-content/book-info.tsx

```
@@ -98,17 +101,62 @@ export default function BookInfo({ className }: { className?: string }) {
-          <Link key={genre.id} href={navigation.genres.bySlug(genre.slug)}>
-            <Badge variant="outline" className="hover:bg-accent">
-              {genre.name}
-            </Badge>
-          </Link>
-        ))}
-      </div>
+      <DottedList
+        className="mt-6 text-xs md:text-sm"
+        items={[
+          advancedGenres.length > 0 ? (
+            <div className="flex flex-wrap items-center gap-2">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)